deployed-contracts/isolated-pools.md

```diff
@@ -229,6 +229,19 @@
   * vUSDC\_Core: [`0x7D8609f8da70fF9027E9bc5229Af4F6727662707`](https://arbiscan.io/address/0x7D8609f8da70fF9027E9bc5229Af4F6727662707)
   * vARB\_Core: [`0xAeB0FEd69354f34831fe1D16475D9A83ddaCaDA6`](https://arbiscan.io/address/0xAeB0FEd69354f34831fe1D16475D9A83ddaCaDA6)
 
+### Liquid Staked ETH Pool
+
+* Comptroller: [`0x52bAB1aF7Ff770551BD05b9FC2329a0Bf5E23F16`](https://arbiscan.io/address/0x52bAB1aF7Ff770551BD05b9FC2329a0Bf5E23F16)
+* NativeTokenGateway: [`0xD1e89806BAB8Cd7680DFc7425D1fA6d7D5F0C3FE`](https://arbiscan.io/address/0xD1e89806BAB8Cd7680DFc7425D1fA6d7D5F0C3FE)
+* Underlying tokens:
+  * WSTETH: [`0x5979D7b546E38E414F7E9822514be443A4800529`](https://arbiscan.io/address/0x5979D7b546E38E414F7E9822514be443A4800529)
+  * WEETH: [`0x35751007a407ca6FEFfE80b3cB397736D2cf4dbe`](https://arbiscan.io/address/0x35751007a407ca6FEFfE80b3cB397736D2cf4dbe)
+  * WETH: [`0x82af49447d8a07e3bd95bd0d56f35241523fbab1`](https://arbiscan.io/address/0x82af49447d8a07e3bd95bd0d56f35241523fbab1)
+* Markets:
+  * vWSTETH\_LiquidStakedETH: [`0x9df6B5132135f14719696bBAe3C54BAb272fDb16`](https://arbiscan.io/address/0x9df6B5132135f14719696bBAe3C54BAb272fDb16)
+  * vWEETH\_LiquidStakedETH: [`0x246a35E79a3a0618535A469aDaF5091cAA9f7E88`](https://arbiscan.io/address/0x246a35E79a3a0618535A469aDaF5091cAA9f7E88)
+  * vWETH\_LiquidStakedETH: [`0x39D6d13Ea59548637104E40e729E4aABE27FE106`](https://arbiscan.io/address/0x39D6d13Ea59548637104E40e729E4aABE27FE106)
```

Comment:
To be consistent with the symbols used in other sections:

```suggestion
  * wstETH: [`0x5979D7b546E38E414F7E9822514be443A4800529`](https://arbiscan.io/address/0x5979D7b546E38E414F7E9822514be443A4800529)
  * weETH: [`0x35751007a407ca6FEFfE80b3cB397736D2cf4dbe`](https://arbiscan.io/address/0x35751007a407ca6FEFfE80b3cB397736D2cf4dbe)
  * WETH: [`0x82af49447d8a07e3bd95bd0d56f35241523fbab1`](https://arbiscan.io/address/0x82af49447d8a07e3bd95bd0d56f35241523fbab1)
* Markets:
  * vwstETH\_LiquidStakedETH: [`0x9df6B5132135f14719696bBAe3C54BAb272fDb16`](https://arbiscan.io/address/0x9df6B5132135f14719696bBAe3C54BAb272fDb16)
  * vweETH\_LiquidStakedETH: [`0x246a35E79a3a0618535A469aDaF5091cAA9f7E88`](https://arbiscan.io/address/0x246a35E79a3a0618535A469aDaF5091cAA9f7E88)
  * vWETH\_LiquidStakedETH: [`0x39D6d13Ea59548637104E40e729E4aABE27FE106`](https://arbiscan.io/address/0x39D6d13Ea59548637104E40e729E4aABE27FE106)
```

Reply:
Fixed, 8db135cb50f38b062efa6eb42b239f48186bb652